scripts/hash-files.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# this script emits as hash for the files listed in $@
+if command -v shasum >/dev/null 2>&1; then
+  cat "$@" | shasum -a 256 -p | cut -d' ' -f1
+elif command -v sha256sum >/dev/null 2>&1; then
+  cat "$@" | sha256sum -p | cut -d' ' -f1
```

Comment:
This might only be Ubuntu 20.04, but it looks like I don't have `--portable` (`-p`) for either shasum or sha256sum. A brief google search makes it looks like it's mainly MacOS.

Reply:
ack, I don't think we actually need this, and it would be easy to revise later.
breadcrumb for the future: thinking is someone on e.g. WSL2 and some weird line ending issue, git on windows supports rewriting line-endings locally (`\n` <=> `\r\n`)